.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  // "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Did you intend to have this commented out in the source-controlled version?

Reply:
Still researching if we need that in place, formatting works as intended without prettier settings being used as default